common/src/main/java/com/wynntils/features/user/ChatTabsFeature.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.google.common.collect.Sets;
+import com.google.gson.reflect.TypeToken;
+import com.wynntils.core.chat.ChatModel;
+import com.wynntils.core.chat.RecipientType;
+import com.wynntils.core.chat.tabs.ChatTab;
+import com.wynntils.core.chat.tabs.ChatTabModel;
+import com.wynntils.core.config.Config;
+import com.wynntils.core.config.TypeOverride;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.managers.Model;
+import com.wynntils.gui.widgets.ChatTabButton;
+import com.wynntils.mc.event.LocalMessageEvent;
+import com.wynntils.mc.event.ScreenInitEvent;
+import com.wynntils.mc.event.ScreenRenderEvent;
+import com.wynntils.wynn.event.ChatMessageReceivedEvent;
+import com.wynntils.wynn.event.WorldStateEvent;
+import com.wynntils.wynn.model.WorldStateManager;
+import java.lang.reflect.Type;
+import java.util.Arrays;
+import java.util.List;
+import net.minecraft.client.gui.screens.ChatScreen;
+import net.minecraftforge.eventbus.api.EventPriority;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class ChatTabsFeature extends UserFeature {
+    public static ChatTabsFeature INSTANCE;
+
+    @Config(visible = false)
+    public List<ChatTab> chatTabs = Arrays.asList(
+            new ChatTab("All", true, null, ".*"),
```

Comment:
I think the regexp here should be `null` as well. `All` is kind of conceptually the tab with no filters whatsoever.

Reply:
Ah, right, it's saved by json for the configuration.

Reply:
> I think the regexp here should be `null` as well. `All` is kind of conceptually the tab with no filters whatsoever.

That would mean I don't have a regex so I don't match anything there? Unless I change null regex to match everything which would mean I can't filter commands from tabs.

Reply:
Yes, I mean that the null regexp should match everything. (And I think I presented a solution for the command replies)